packages/core/forms/src/composables/useAbstractFields.ts

```diff
@@ -26,9 +26,10 @@ interface AbstractFieldParams {
     errors: any[]
     field: Record<string, any>
   }) => void
+  externalValidator?: (value: any, field: any, model: any) => string[] | Promise<string[]>
 }
 
-export default function useAbstractFields(formData: AbstractFieldParams) {
+export default function useAbstractFields<T = any>(formData: AbstractFieldParams) {
```

Comment:
Would `unknown` be a better option here?

Reply:
I don't think so. Using `any` here gives us maximum compatibility with the existing code, however using `unknown` here will break everything

Reply:
Noted